### PR TITLE
changed documentation in radix.cpp

### PR DESCRIPTION
**Pull request by jvmolu**
Anmol Singhal
11940140
anmolsinghal@iitbhilai.ac.in

### CODE REVIEW

File: Sorting/radix.cpp [hunk 1]
```diff
@@ -1,14 +1,16 @@
 #include "sorting.hpp"
 
 /**
- * Documentation should go here
+ * It returns the value of base raised to the power exp.
```
Comment (shashwatj07) [inline Sorting/radix.cpp]: ~It~ Function

**Review by shashwatj07 — CHANGES_REQUESTED** [review]
a minor item

File: Sorting/radix.cpp [hunk 2]
```diff
@@ -1,14 +1,16 @@
 #include "sorting.hpp"
 
 /**
- * Documentation should go here
+ * pow function returns the value of base raised to the power exp.
```
Comment (shashwatj07) [inline Sorting/radix.cpp]: ```Function returns the value of base raised to the power exp.```

**Review by shashwatj07 — CHANGES_REQUESTED** [review]

**Comment by jvmolu** [comment]
@shashwatj07 Done

**Review by yatendra1999 — APPROVED** [review]
LGTM

**Review by shashwatj07 — APPROVED** [review]

**Pull request merged**